### PR TITLE
OS-7698 Add mkzpool -e for creating encrypted zpools

### PR DESCRIPTION
**Pull request by jasonbking**

### CODE REVIEW

File: src/node_modules/zfs.js [hunk 1]
```diff
@@ -124,22 +125,35 @@ zpool.status = function (pool, callback) {
  * zpool create command, including log devices, cache devices, and hot spares.
  * The input is an object of the form produced by the disklayout library.
  */
-zpool.create = function (pool, config, force, callback) {
+zpool.create = function (pool, config, force, encrypt, callback) {
+    var cmd;
```
Comment (mgerdts) [inline src/node_modules/zfs.js]: Do we now have a weird mixture of spaces and tabs?

File: src/node_modules/zfs.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2013 Joyent, Inc.  All rights reserved.
+ * Copyright 2019 Joyent, Inc.  All rights reserved.
```
Comment (mgerdts) [inline src/node_modules/zfs.js]: `Copyright 2019 Joyent, Inc.`

File: man/man/man1m/mkzpool.1m [hunk 3]
```diff
@@ -0,0 +1,41 @@
+.Dd November 13, 2019
```
Comment (bahamat) [inline man/man/man1m/mkzpool.1m]: Date needs to be updated

Comment (jasonbking) [reply man/man/man1m/mkzpool.1m]: The date on man pages is usually the date of last modification (that's how it is for all of the illumos man pages).

Comment (bahamat) [reply man/man/man1m/mkzpool.1m]: Discussed with @jasonbking, this is accurate as to when the change was authored, so I call it good.

**Pull request merged**